src/test/resources/application.conf

```diff
@@ -17,9 +17,13 @@ env: local
 local {
   services {
     host: "http://localhost"
-    auth-login-stub {
-      port: 9949
-      productionRoute = "/auth-login-stub"
+    auth-login-stub-reg {
+      port: 14000
+      productionRoute = ""
+    }
+    auth-login-stub-ret {
```

Comment:
Let's rename these - they are no longer the auth-login-stub .. they are the start pages for the relevant services

so they should be:

`economic-crime-levy-registration-frontend`
`economic-crime-levy-returns-frontend`

Reply:
done